lib/nostrum/struct/message/component.ex

```diff
@@ -194,6 +204,14 @@ defmodule Nostrum.Struct.Message.Component do
   @typedoc since: "0.5.1"
   @type value :: String.t() | nil
 
+  @typedoc """
+  Child component for labels.
+
+  Only present for labels.
+  """
+  @typedoc since: "NEXTVERSION"
+  @type component :: [t()]
```

Comment:
This should be a single child then, no? Instead of a list.

```suggestion
  @type component :: t()
```

Reply:
Haha, absolutely. All that talk about it being a single one and I forgot it myself right away. :) Thank you, should be fixed.